controllers/useraccount/useraccount_controller.go

```diff
@@ -484,6 +500,93 @@ func (r *Reconciler) deleteUser(ctx context.Context, userAcc *toolchainv1alpha1.
 	return true, nil
 }
 
+// Returns `true` if the associated resources (configMap, role and role-binding) created for a user by console is deleted, `false` otherwise with the underlying error.
+// This function only looks for these resources in the namespace - openshift-console-user-settings -
+func (r *Reconciler) deleteUserResources(ctx context.Context, userID string) (bool, error) {
```

Comment:
Just to make it clear that it's not the SSO `UserID` but the `User.metadata.uid`.
```suggestion
func (r *Reconciler) deleteUserResources(ctx context.Context, userUID string) (bool, error) {
```

Reply:
done in [de45e6b](https://github.com/codeready-toolchain/member-operator/pull/568/commits/de45e6b31fcd13fe55a985f7614164dfb008a934)